src/v2/Components/FlashBanner/index.tsx

```diff
@@ -17,9 +18,9 @@ interface FlashBannerProps {
 
 /**
  * The component responsible for selecting a determining and displaying a flash message
- *
  */
 export const FlashBanner: React.FC<FlashBannerProps> = props => {
+  if (isServer) return null
```

Comment:
this breaks the abiility for other code to share this banner, in which case we might want to SSR render. Have we encounted a specific instance where this line is needed?